engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -1,28 +1,29 @@
 package cromwell.engine.workflow
 
-import java.time.OffsetDateTime
-
 import akka.actor.FSM.{CurrentState, SubscribeTransitionCallBack, Transition}
 import akka.actor._
 import akka.event.Logging
 import akka.routing.FromConfig
 import com.typesafe.config.{Config, ConfigFactory}
-import cromwell.core.{WorkflowId, WorkflowSourceFiles}
-import cromwell.database.obj.WorkflowMetadataKeys
+import cromwell.core.WorkflowId
 import cromwell.engine._
 import cromwell.engine.workflow.WorkflowActor._
 import cromwell.engine.workflow.WorkflowManagerActor._
 import cromwell.engine.workflow.lifecycle.CopyWorkflowLogsActor
 import cromwell.services.MetadataServiceActor._
-import cromwell.services.{MetadataEvent, MetadataKey, MetadataValue, ServiceRegistryClient}
+import cromwell.services.ServiceRegistryClient
 import cromwell.webservice.CromwellApiHandler._
 import lenthall.config.ScalaConfig.EnhancedScalaConfig
 
 import scala.concurrent.duration._
 import scala.concurrent.{Await, Promise}
 import scala.language.postfixOps
+import scalaz.NonEmptyList
 
 object WorkflowManagerActor {
+  val DefaultMaxWorkflowsToRun = 5000
```

Comment:
I can't imagine a universally sane default for the max number of workflows to run, but I'm not sure if it's better to have the default be too low or too high.  For Firecloud this number might be fine but it's hard to imagine GOTC Cromwell being able to survive 5000 concurrent workflows (though I'd love to be proven wrong about that 😄 ).